src/components/author-primitives/packer/badge/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
Author primitives for `packer` weren't up to date, so I brought in the latest ones from the `packer` repo's `website` directory.